prime-dotnet-webapi/Services/Rules/AutomaticAdjudicationRules.cs

```diff
@@ -155,12 +155,12 @@ public class RequestingRemoteAccessRule : AutomaticAdjudicationRule
     {
         public override Task<bool> ProcessRule(Enrollee enrollee)
         {
-            if (enrollee.RequestingRemoteAccess)
+            if (enrollee.EnrolleeRemoteUsers.Count > 0)
             {
                 enrollee.AddReasonToCurrentStatus(StatusReasonType.RequestingRemoteAccess);
             }
 
-            return Task.FromResult(!enrollee.RequestingRemoteAccess);
+            return Task.FromResult(!(enrollee.EnrolleeRemoteUsers.Count > 0));
```

Comment:
Looks a bit nicer to return `Task.FromResult(true)` here and `Task.FromResult(false)` inside the if statement